packages/client/src/styles/globals.css

```
@@ -0,0 +1,226 @@
+ *
+ * Dark theme inspired by LCARS (Star Trek) with orange primary.
+ *
+ * Architecture:
+ *   Components reference semantic tokens: bg-[var(--color-card-bg)]
+ *   Semantic tokens alias primitives: var(--color-gray-850)
+ *   This indirection means swapping themes only touches Tier 2.
+ */
+
+@import 'tailwindcss';
+
+@source '../components';
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)